src/plugins/analysis/qemu_exec/view/qemu_exec.html

```diff
@@ -11,16 +11,16 @@
     </style>
 {% endblock %}
 
+{% set analysis = firmware.processed_analysis[selected_analysis]['result'] %}
```

Comment:
I think it is unfortunate to call the result dict `analysis` in this and all other occurrences. I'd expect to, e.g., uniformly access the summary across all plugins as follows:

```jinja
{% analysis['summary'] %}
```